h/accounts/test/views_test.py

```diff
@@ -79,9 +80,14 @@ def mock_flash_function():
                                 return_value=None)
 
 
-@pytest.mark.usefixtures('routes_mapper')
 class TestAuthController(object):
 
+    @pytest.fixture(autouse=True)
+    def routes(self, config):
+        config.add_route('forgot_password', '/forgot')
+        config.add_route('index', '/index')
+        config.add_route('stream', '/stream')
```

Comment:
> Any reason you used a class-level autouse fixture, rather than usefixtures?

No particular reason. It just seems to me to be a more obvious way to use fixtures than having to declare the fixture in one place and then declare that it is used later.

Happy to switch to using `usefixtures` everywhere.